dbquery/custom-query/api.graphql

```diff
@@ -0,0 +1,42 @@
+# This example shows how @dbquery is configured for custom SQL queries with Customer data.
+
+"""
+Sample Customer type.
+"""
+type Customer {
+  id: ID!
+  name: String
+  email: String
+}
+
+"""
+`Customer` queries a mock `Customer` object from a demo PostgreSQL database.
+The data is fetched using custom SQL queries.
+
+Table structure for 'customer':
+
+  CREATE TABLE customer (
+    id SERIAL PRIMARY KEY,          -- Unique identifier with sequence
+    name CHARACTER(50) NOT NULL,    -- Customer's name, max 50 characters
+    email CHARACTER(50) NOT NULL,   -- Customer's email, max 50 characters, must be unique
+    CONSTRAINT customer_email_key UNIQUE (email)  -- Unique constraint on email
+  );
+
+"""
+type Query {
+  # Fetches all customers.
+  customers: [Customer]
+    @dbquery(
+      query: "SELECT id, name, email FROM customer",
+      type: "postgresql",
+      configuration: "postgresql_config"
+    )
+
+  # Fetches a customer by their ID.
+  customer(id: Int!): Customer
```

Comment:
Expand comments here to describe the mapping between field arguments and parameter markers. 
Snippets are a teaching aid, so it helps to describe details of  the directive usage.